lib/logstash-input-google_pubsub_jars.rb

```diff
@@ -51,4 +51,4 @@
 require_jar('org.codehaus.mojo', 'animal-sniffer-annotations', '1.23')
 require_jar('org.conscrypt', 'conscrypt-openjdk-uber', '2.5.2')
 require_jar('org.threeten', 'threetenbp', '1.6.8')
-require_jar('org.logstash.inputs', 'logstash-input-google_pubsub', '1.3.0')
+require_jar('org.logstash.inputs', 'logstash-input-google_pubsub', '1.4.0')
```

Comment:
Not following this change? Is this something that was missed previously?

Reply:
Yes, looks like it - this is automatically generated during the building of the jar. I assume whoever made the last update didn't build locally...